web/src/main/angular/src/app/core/components/apdex-score/apdex-score.component.ts

```diff
@@ -0,0 +1,80 @@
+import { Component, ComponentFactoryResolver, Injector, Input, OnInit, SimpleChanges } from '@angular/core';
```

Comment:
Add `OnChanges` lifecycle
```suggestion
import { Component, ComponentFactoryResolver, Injector, Input, OnInit, SimpleChanges, OnChanges } from '@angular/core';
```